easybuild/framework/easyconfig/format/one.py

```diff
@@ -150,12 +151,14 @@ def _reformat_line(self, param_name, param_val, outer=False, addlen=0):
 
         elif outer:
             # only reformat outer (iterable) values for (too) long lines (or for select parameters)
-            if isinstance(param_val, (list, tuple, dict)) and ((len(param_val) > 1 and line_too_long) or forced):
+            if isinstance(param_val, (list, tuple, dict)) and ((len(param_val) > 1 or line_too_long) or forced):
```

Comment:
This one and -> or i don't fully understand. Could you explain it a bit?

Reply:
I think this was a logic error...

It should always do one-element-per-line for lists with more than 1 element or very long lines, at least for "outer" iterables (so not necessarily for a list in a dict, like `sanity_check_paths`).